grafana/dashboards/efficiency.json

```diff
@@ -551,7 +554,7 @@
             "properties": [
               {
                 "id": "displayName",
-                "value": "Efficiency"
+                "value": "Driving Efficiency"
```

Comment:
Renamed to Driving Efficiency across all dashboards - Efficiency is used in three different ways:

- Charging Efficiency
- Driving Efficiency
- Vehicle Efficiency -> Ideal/Rated Consumption -> I've not changed it, still Efficiency